index.js

```diff
@@ -46,7 +47,7 @@ var gs = {
       stream.write({
         cwd: opt.cwd,
         base: basePath,
-        path: path.resolve(opt.cwd, filename),
+        path: filename,
```

Comment:
yeah, this seems to be doing nothing currently too so it was probably just an extra call